Tests/CreateAPITests/Expected/edgecases-yaml-config/Sources/Entities.swift

```diff
@@ -281,6 +291,21 @@ public struct Animal: Codable {
     }
 }
 
+public struct Image: Codable {
+    public var identifier: AnyJSON
+    public var url: AnyJSON
```

Comment:
Do you need to do anything to re-generate this? Looking at the schema, I think these should be `String` and not `AnyJSON` but I can't figure out why 🤔